api/column_family.cc

```diff
@@ -335,13 +335,13 @@ void set_column_family(http_context& ctx, routes& r) {
     cf::get_memtable_columns_count.set(r, [&ctx] (std::unique_ptr<request> req) {
         return map_reduce_cf(ctx, req->param["name"], 0, [](column_family& cf) {
             return cf.active_memtable().partition_count();
```

Comment:
What's not expected in the way it works now?

Reply:
2147483648 + 1 = -2147483647 and that's probably not the intention of this code

Reply:
Ok. It's nice to be explicit about it, and also file an issue and add Fixes #nnnn.

Reply:
done